Cubical/Foundations/Path.agda

```diff
@@ -188,6 +188,65 @@ Square≃doubleComp : {a₀₀ a₀₁ a₁₀ a₁₁ : A}
                     → Square a₀₋ a₁₋ a₋₀ a₋₁ ≃ (a₋₀ ⁻¹ ∙∙ a₀₋ ∙∙ a₋₁ ≡ a₁₋)
 Square≃doubleComp a₀₋ a₁₋ a₋₀ a₋₁ = transportEquiv (PathP≡doubleCompPathˡ a₋₀ a₀₋ a₁₋ a₋₁)
 
+-- Flipping a square in Ω²A is the same as inverting it
+sym≡flipSquare : ∀ {ℓ} {A : Type ℓ} {x : A} (P : Square (refl {x = x}) refl refl refl)
```

Comment:
Why quantify over `ℓ` and `A` when they are variables in the file?

Reply:
Because copy-paste;-) Fixed now.